dash/development/base_component.py

```diff
@@ -4,8 +4,13 @@
 import inspect
 import abc
 import sys
+import pprint
+from textwrap import dedent
+
 import six
 
+import dash.exceptions
```

Comment:
Shouldn't it be `import ..exceptions` to keep consistent with the relative imports ?

Reply:
Yeah I'll change, that looks better